conf/messages

```diff
@@ -1289,20 +1296,26 @@ pages.CoverSheet.vat1TRLink                        = VAT1TR form
 
 ## Northern Ireland Protocol
 pages.northernIrelandReceiveGoods.heading                = Does the business expect to receive goods in Northern Ireland from an EU country?
-pages.northernIrelandReceiveGoods.hint                   = What is the value of these goods?
-nip.receiveGoods.missing                                 = Select yes if the business expects to receive goods in Northern Ireland from an EU country
-validation.northernIrelandReceiveGoods.missing           = Enter the value of these goods
-validation.northernIrelandReceiveGoods.invalid           = Enter the value of these goods using characters 0 to 9
+pages.northernIrelandReceiveGoods.label                  = What is the value of these goods?
+pages.northernIrelandReceiveGoods.hint                   = For example, £600 or £9000
+nip.receiveGoods.missing                                 = Select yes if the business expects to receive goods in Northern Ireland from an EU country in the next 12 months
+validation.northernIrelandReceiveGoods.missing           = Enter the estimated value of the goods
+validation.northernIrelandReceiveGoods.invalid           = Enter the estimated value of the goods in the correct format
+validation.northernIrelandReceiveGoods.commasNotAllowed  = The estimated value of the goods cannot include commas
 
 # NORTHERN IRELAND PROTOCOL
 
 nip.sellOrMove.heading                                   = Does the business expect to do any of the following in the next 12 months?
 nip.sellOrMove.para                                      = Tell us if the business will do any of the following:
 nip.sellOrMove.bullet1                                   = sell goods located in Northern Ireland to any country
 nip.sellOrMove.bullet2                                   = sell or move goods from Northern Ireland to an EU country
-nip.sellOrMove.hint                                      = What is the value of these goods?
-nip.error.missing                                        = Select yes if the business will do any of the activities in the next 12 months
-validation.sellOrMoveNip.missing                         = Enter the value of these goods
+nip.sellOrMove.label                                     = What is the value of these goods?
```

Comment:
What is the *estimated value of these goods?

Reply:
We are changing hint and error messages under this ticket, that one is a content change so not changing it here.